mlir/lib/Conversion/LLVMCommon/TypeConverter.cpp

```diff
@@ -86,7 +86,15 @@ LLVMTypeConverter::LLVMTypeConverter(MLIRContext *ctx,
 
     if (type.isIdentified()) {
       auto convertedType = LLVM::LLVMStructType::getIdentified(
-          type.getContext(), ("_Converted." + type.getName()).str());
+          type.getContext(), ("_Converted_" + type.getName()).str());
```

Comment:
This change is a bit strange. Is that some rebase artifact? I think these lines were deleted by another PR.